packages/core/src/components/graphs/scatter.ts

```diff
@@ -57,6 +57,44 @@ export class Scatter extends Component {
 		this.addEventListeners();
 	}
 
+	isThresholdsAnomaly(datum: any, index: number) {
```

Comment:
maybe a comment here as to what this means, and potentially a more readable name